tests/test_claude_wrapper_hooks.py

```
@@ -654,6 +665,107 @@ def test_stale_socket_skips_hook_injection(failures: list[str]) -> None:
+            node_options.startswith("--require="),
+            f"#3463 tmpdir wipe: expected wrapper to set NODE_OPTIONS --require, got {node_options!r}",
+            failures,
+        )
+
+        guard_dir_tmpdir = tmp_dir / "cmux-claude-node-options"
+        if guard_dir_tmpdir.exists():
+            shutil.rmtree(guard_dir_tmpdir, ignore_errors=True)
+
+        node_env = os.environ.copy()
+        node_env["NODE_OPTIONS"] = node_options
+        result = subprocess.run(
```

Ruff (0.15.12)

[error] `subprocess` call: check for execution of untrusted input

(S603)